concepts/durations/ventdurations.sql

```diff
@@ -0,0 +1,97 @@
+DROP TABLE IF EXISTS `physionet-data.mimic_derived.ventdurations`;
+CREATE TABLE IF NOT EXISTS `physionet-data.mimic_derived.ventdurations` as
```

Comment:
remove the drop/create statements - this is handled in the (to be uploaded) shell script now, so just the SQL is needed.

Reply:
Done